src/frontend/main/src/routes.jsx

```diff
@@ -16,6 +20,22 @@ const routes = createBrowserRouter(
                     path: "/",
                     element: <Home />,
                 },
+                {
+                    path: "/tabbed",
+                    element: <Tabbed />,
+                },
+                {
+                    path: "/login",
+                    element: <Login />,
+                },
+                {
+                    path: "/signup",
+                    element: <Create />,
+                },
+                {
+                    path: "/recoveraccess",
+                    element: <Forgot />,
```

Comment:
I would be more explicit with the component's name here, for clarity and consistency.

Create -> SignUp
Forgot -> RecoverAccess

Reply:
thank you, Emilio  for the useful feedback..  i had noticed this.  it seems best is when Create -> path; /create   ,,